scripts/init-pd-topo.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python2
+
+from __future__ import print_function, \
+    unicode_literals
+
+import urllib
+import urllib2
+import base64
+import json
+import argparse
+
+ComponentToRegister = ('alertmanager', 'tidb', 'grafana', 'tikv', 'pd')
+
+
+def parse_opts():
+    """
+    parse_opts parse the input of involved components and pd address.
+    """
+    parser = argparse.ArgumentParser(description="Parse output.")
+    # pd is involved because we need to send http request
+    for target in ComponentToRegister:
+        parser.add_argument("--{}".format(target),
+                            help="the address list of {}".format(target))
+    args, unknown = parser.parse_known_args()
+    return args
+
+
+def etcd_write(etcd_url, key, value):
+    encoded_key = base64.b64encode(key)
+    encoded_value = base64.b64encode(value)
+    data = json.dumps({
+        "key": encoded_key,
+        "value": encoded_value,
+    })
+    req = urllib2.Request('http://' + etcd_url + '/v3/kv/put',
```

Comment:
Will this break TLS deployment?

Reply:
At present, this script does not support `TLS`, and it needs to be optimized.

Reply:
@breeswish @liubo0127  maybe we have 2 ways:
* install etcd in pip and get it.
* Using https.